scripts/verify_tests_on_master.bat

```diff
@@ -1,5 +1,21 @@
 @ECHO ON
 
+:: Default values.
```

Comment:
nit: should we stick to one comments syntax (:: vs REM)?

Reply:
I prefer :: as the comments are not included in the log, but because the rest of the batch scripts use REM I will switch to REM to maintain consistency.

Reply:
I'd be ok with switching everything to `::` if you wanted to.

Reply:
I'll do a follow up PR to switch it over then, that should be an easier diff to read :)